src/runner/nunit.runner.Droid/nunit.runner.Droid.nuget.props

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
NuGet recommends that *.nuget.props is gitignored. 😄 

https://docs.microsoft.com/en-us/nuget/consume-packages/packages-and-source-control#omitting-packages-with-git